app/webpacker/components/TheAnnotator.vue

```diff
@@ -27,7 +27,6 @@
   <ContextMenu ref="linkMenu"
                :closeOnClick="false">
     <form @submit.prevent="submit('link', content)"
-          @keyup.esc="close"
```

Comment:
This wasn't actually being used. the scrollEventListener in ContextMenu automatically looks for focus changes and calls the close method